pkg/controller/persistentvolume/persistentvolume_controller.go

```diff
@@ -73,6 +73,30 @@ const annBoundByController = "pv.kubernetes.io/bound-by-controller"
 // provisioning to choose a particular storage class (aka profile).
 const annClass = "volume.alpha.kubernetes.io/storage-class"
 
+// This annotation is added to a PV that has been dynamically provisioned by
+// Kubernetes. It's value is name of volume plugin that created the volume.
+// It serves both user (to show where a PV comes from) and Kubernetes (to
+// recognize dynamically provisioned PVs in its decissions).
+const annDynamicallyProvisioned = "pv.kubernetes.io/provisioned-by"
+
+// Name of a tag attached to a real volume in cloud (e.g. AWS EBS or GCE PD)
+// with namespace of a persistent volume claim used to create this volume.
+const cloudVolumeCreatedForClaimNamespaceTag = "kubernetes.io/created-for/pvc/namespace"
+
+// Name of a tag attached to a real volume in cloud (e.g. AWS EBS or GCE PD)
+// with name of a persistent volume claim used to create this volume.
+const cloudVolumeCreatedForClaimNameTag = "kubernetes.io/created-for/pvc/name"
+
+// Name of a tag attached to a real volume in cloud (e.g. AWS EBS or GCE PD)
+// with name of appropriate Kubernetes persistent volume .
+const cloudVolumeCreatedForVolumeNameTag = "kubernetes.io/created-for/pv/name"
```

Comment:
this is not a valid kube annotation name.  can we at least be consistent with that?  pv.kubernetes.io/created-for-pv or something?